data/MuthoniMN.json

```
@@ -0,0 +1,29 @@
+{
+  "name": "Michelle Ndiangui",
+  "bio": "Full-Stack Software Engineer | MERN | Technical Writer | Computer Science Student at Riara University",
+  "links": [
+    {
+      "name" : "Visit My Web Development Blog",
+      "url": "https://mitchcodes.hashnode.dev/",
+      "icon": "FaLink",
+      "color": "#FFCAE9"
+    }
+  ],
+  "socials": [
+    {
+      "icon": "FaGithub",
+      "url" : "https://github.com/MuthoniMN"
+    },
+    {
+      "icon": "FaLinkedin",
+      "url" : "https://www.linkedin.com/in/michelle-ndiangui/"
+    },
+    {
+      "icon": "FaTwitter",
+      "url" : "https://twitter.com/codebymitch"
+    },
+    {
+      "icon": "FaInstagram",
+      "url" : "https://www.instagram.com/codebymitch/"
+    }
+  ]
```

Please add the missing closing braces `}` in the end.
```suggestion
  ]
}
```

---

I've added the closing brace. Please review my code again